hybrid/compartment_examples/inter_comp_call/secure-update_ddc/switch_compartment-update_ddc.s

```diff
@@ -108,7 +108,8 @@ clean:
     mov x18, x18
     // x19-x29 are callee-saved, but only the lower 64 bits.
     mov x19, x19
-    mov x20, x20
+    // Simulate leaking adress of switcher DDC
+    //mov x20, x20
```

Comment:
Should this be deleted?

Reply:
Actually no; I think the change on lines 54-56 should be reverted, and this change kept in to emphasise that we use that register to emulate the address leak.

Reply:
OK, maybe push a change for that (not force push as @jacobbramley might be reviewing as we speak)?

Reply:
You can force-push; I'm finishing something off and will probably get here after lunch.

Reply:
Done in https://github.com/capablevms/cheri-examples/pull/57/commits/44bb3d8a3272205682b9d860f01a90b36e85092e.

Reply:
I'd rather avoid force pushing, but those force pushes were done as we wanted the Github UI to comply, and that was the only way to do so.